### PR TITLE
Update writing-a-great-proposal.md

### PR DESCRIPTION
**Pull request by dewanshrawat15**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: _faq/writing-a-great-proposal.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ lot of helpful feedback from us!
 
 #### Writing the proposal
 
-Please follow the **application template** in the faq. All fields are required if not explicitly indicated otherwise. While following the application template you can consider doing an illustration of your proposed solution as it makes the application much more intuitive.
+Please follow the **application template[application template](https://raw.githubusercontent.com/coala/projects/master/_faq/application-template.md)** in the faq. All fields are required if not explicitly indicated otherwise. While following the application template you can consider doing an illustration of your proposed solution as it makes the application much more intuitive.
```
Comment (srivama) [inline _faq/writing-a-great-proposal.md]: Mentioning application template in `[]` will mention it on the website once, no need to write it once again before `[ `

Comment (srivama) [inline _faq/writing-a-great-proposal.md]: When the user clicks on the anchor, he should be directed to the above appication template tab on the same webpage, not the .md file. Please click on netlify preview to have a look at the changes whenever you push :wink:

**Comment by li-boxuan** [comment]
Which issue are you solving? You need to mention that in your commit message and open comment, i.e. https://github.com/coala/projects/pull/677#issue-225406603

**Comment by dewanshrawat15** [comment]
I spent a lot of time researching but still haven't been able to come up with a solution as to how to fix this bug. Can someone please help me out?

**Comment by li-boxuan** [comment]
@dewanshrawat15 What bug are you aiming to solve? Please mention that in the open comment and your commit message. See my previous message https://github.com/coala/projects/pull/677#issuecomment-432894309

**Comment by srivama** [comment]
One approach you can consider is, adding an `id` to the `div` containing the application template and using that to point the link that you'll be adding to the above div. This will scroll the entire page upwards, setting the view to the first question. Next, you'll have to figure out how on clicking the link, you'll expand the `collapsed div` of the first answer.

**Pull request closed without merging**